frontend/src/app/modules/auth/register-form/register-form.component.html

```diff
@@ -2,46 +2,90 @@
+                Workspace is a required property
+            </small>
+            <small *ngIf=" registerForm.controls['workspaceName'].touched && registerForm.controls['workspaceName'].errors && registerForm.controls['workspaceName'].errors['minlength']">
+                Workspace must be at least 3 characters long
+            </small>
+            <small *ngIf=" registerForm.controls['workspaceName'].touched && registerForm.controls['workspaceName'].errors && registerForm.controls['workspaceName'].errors['maxlength']">
+                Workspace must be no more than 30 characters long
+            </small>
+        </div>
+
         <p class="input-label">Email</p>
-        <mat-form-field class="input-form" appearance="fill" color="warn" floatLabel="never" hideRequiredMarker="true">
-            <input type="email" matInput placeholder="user@gmail.com" formControlName="email" required>
+        <mat-form-field class="input-form" appearance="fill" floatLabel="never" hideRequiredMarker="true">
+            <input type="email" matInput placeholder="user@gmail.com" formControlName="email">
         </mat-form-field>
+        <div class="errors">
+            <small *ngIf=" registerForm.controls['email'].touched && registerForm.controls['email'].errors && registerForm.controls['email'].errors['required']">
+                Email is a required property
+            </small>
+            <small *ngIf=" registerForm.controls['email'].touched && registerForm.controls['email'].errors && registerForm.controls['email'].errors['pattern']">
+                Invalid email
+            </small>
+        </div>
+
         <p class="input-label">Password</p>
-        <mat-form-field class="input-form" appearance="fill" color="warn" hideRequiredMarker="true">
+        <mat-form-field class="input-form" appearance="fill" hideRequiredMarker="true">
             <input [type]="hidePass ? 'password' : 'text'" matInput minlength="8" placeholder="password"
-                formControlName="password" required>
+                formControlName="password">
             <mat-icon class="password-icon" matSuffix (click)="hidePass = !hidePass">{{hidePass ? 'visibility' :
                 'visibility_off'}}
             </mat-icon>
         </mat-form-field>
+        <div class="errors">
+            <small *ngIf=" registerForm.controls['password'].touched && registerForm.controls['password'].errors && registerForm.controls['password'].errors['required']">
+                Password is a required property
+            </small>
+            <small *ngIf=" registerForm.controls['password'].touched && registerForm.controls['password'].errors && 
+            ( registerForm.controls['password'].errors['pattern'] || 
+              registerForm.controls['password'].errors['minlength'] ||
+              registerForm.controls['password'].errors['maxlength'] )">
```

Comment:
```suggestion
            <small *ngIf=" registerForm.controls['password'].touched && registerForm.controls['password'].errors">
```
Correct me if I'm wrong